src/EditorFeatures/Core/Extensibility/NavigationBar/NavigationBarActionlessItem.cs

```diff
@@ -1,36 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
all the individual types got merged into one effective 'union' type.  I don't love it, but it's an easy mechanism to serialize from OOP to VS.  We could have the flattened union form for serialization, and the individual forms for creation/consumption, but i didn't think it really helped out, so i just removed them all.